src/go-postgres/devcontainer-template.json

```diff
@@ -7,19 +7,18 @@
     "publisher": "Dev Container Spec Maintainers",
```

Comment:
Can we bump the major version? Thanks! 

https://github.com/devcontainers/templates/blob/c2361536659318af8637b8726e732c95b039f5e6/src/go-postgres/devcontainer-template.json#L3